tools/experiments/experiments.js

```diff
@@ -268,41 +281,59 @@ function updateExperimentRow(experiment) {
  * @return {boolean}
  */
 function isExperimentOn_(id) {
+  // TODO(#25205): remove this once the CDN stops supporting the AMP_CANARY
+  // cookie.
+  const optInCookieValue =
+    getCookie(window, '__Host-AMP_OPT_IN') || getCookie(window, 'AMP_CANARY');
   switch (id) {
     case EXPERIMENTAL_CHANNEL_ID:
-      return getCookie(window, 'AMP_CANARY') == AMP_CANARY_COOKIE.CANARY;
+      return [
+        AMP_OPT_IN_COOKIE.EXPERIMENTAL,
+        _LEGACY_AMP_OPT_IN_COOKIE.EXPERIMENTAL,
+      ].includes(optInCookieValue);
     case BETA_CHANNEL_ID:
-      return getCookie(window, 'AMP_CANARY') == AMP_CANARY_COOKIE.RC;
+      return [AMP_OPT_IN_COOKIE.BETA, _LEGACY_AMP_OPT_IN_COOKIE.BETA].includes(
+        optInCookieValue
+      );
     case RTV_CHANNEL_ID:
-      return RTV_PATTERN.test(getCookie(window, 'AMP_CANARY'));
+      return RTV_PATTERN.test(optInCookieValue);
     default:
       return isExperimentOn(window, /*OK*/ id);
   }
 }
 
 /**
- * Opts in to / out of the "canary" or "rc" runtime types by setting the
- * AMP_CANARY cookie.
- * @param {string} cookieState One of AMP_CANARY_COOKIE.{DISABLED|CANARY|RC} or
- *   a 15-digit RTV.
+ * Opts in to / out of the "beta" or "experimental" runtime types by setting the
+ * __Host-AMP_OPT_IN cookie.
```

Comment:
Comment doesn't mention RTV. Also, let's say "channel" instead of "type" because our docs use that word.

How about the below?

> "beta" or "experimental" channels or a specific RTV or by setting the...

Reply:
Done